app/presenters/publishing_api/worldwide_organisation_presenter.rb

```diff
@@ -59,7 +59,7 @@ def description
     end
 
     def body
-      Whitehall::GovspeakRenderer.new.govspeak_to_html(item.body) || ""
+      Whitehall::GovspeakRenderer.new.govspeak_edition_to_html(item.about_us) || ""
```

Comment:
Good idea removing the ambiguity of the `body` vs `about_us` here 👍.

Reply:
Credit to @jkempster34 :)